.github/workflows/tests.yml

```diff
@@ -28,7 +28,7 @@ jobs: # Docs: <https://git.io/JvxXE>
         setup: [basic, lowest]
         laravel: [default]
         coverage: [yes]
-        php: ['7.3', '7.4', '8.0']
+        php: ['8.0']
```

Comment:
what about 8.1, 8.2 versions?

Reply:
Alexandr Gomzyakov made pull request for this: https://github.com/avto-dev/amqp-rabbit-laravel-queue/pull/25